### ISSUE TITLE
Upgrade QEMU on Windows

### ISSUE THREAD

**Issue by stlankes** [hermit-os/hermit-rs#327]
Using QEMU 2022.4.19 from choco fails silently: https://github.com/hermitcore/rusty-hermit/runs/7109382481

**Comment by stlankes** [comment]
Has someone a Windows setup to test it on a local machine?

**Comment by stlankes** [comment]
Ok, but we still use the old version of Qemu. Or do I miss-understand something?

**Comment by mkroening** [comment]
Yes, but that might not be an issue with our code. Historically, some QEMU versions on choco just don't work for us. A new version on choco might fix this issue. That's not sure though. We can use this issue to track this though. :+1:

**Comment by mkroening** [comment]
Closing since we don't run CI on Windows since https://github.com/hermit-os/hermit-rs/commit/c175bffab2b3f03884dc9f0f6239a7e043abbb9f. We are sucessfully running Windows CI in [`loader`](https://github.com/hermit-os/loader) and [`hermit-rs-template`](https://github.com/hermit-os/hermit-rs-template).
